index.html

```diff
@@ -0,0 +1,225 @@
+      </div>
+
+
+      <div class="form-group">
+        <p>Would you recommend freeCodeCamp to a friend?</p>
+           <label>
+        <input name= "user-recommend" type="radio" value="Definetely" class="input-radio" checked />
+              Definetely
+            </label>
+              
+            <label>
+              <input name= "user-recommend" type="radio" value="Definetely" class="input-radio" checked />
+                    Maybe
+                  </label>
+
+                  <label>
+                    <input name= "user-recommend" type="radio" value="Definetely" class="input-radio" checked />
+                          Not sure
+                        </label>
+
+
+      </div>
+
+      <div class="form-group">
+        <p>What is your favorite feature of freeCodeCamp?</p>
+        <select>
+          <option>select an option</option>
+          <option>Challenges</option>
+          <option>Projects</option>
+          <option>Community</option>
+          <option>Open source</option>
+        </select>
+
+      </div>
+
+      <div class="form-group">
+        <p>What would you like to see improved? </p>
+        <span class="clue"> (Check all that apply)</span>
+
+        <label>
+          
+          <input type="checkbox" value="front end project" class="input-checkbox"/>
+          Front end project
+
+        </label>
+
+        <label>
+          
+          <input type="checkbox" value="front end project" class="input-checkbox"/>
+          Backend-project
+
+        </label>
+
+        <label>
+          
+          <input type="checkbox" value="front end project" class="input-checkbox"/>
+          Data visualization
+
+        </label>
+
+        <label>
+          
+          <input type="checkbox" value="front end project" class="input-checkbox"/>
+          challenges
+
+        </label>
+
+        <label>
+          
+          <input type="checkbox" value="front end project" class="input-checkbox"/>
+          Open Source Community
+
+        </label>
+
+        <label>
+          
+          <input type="checkbox" value="front end project" class="input-checkbox"/>
+          Gitter help rooms
+
+        </label>
+
+        <label>
+          
+          <input type="checkbox" value="front end project" class="input-checkbox"/>
+          videos
+
+        </label>
+
+        <label>
+          
+          <input type="checkbox" value="front end project" class="input-checkbox"/>
+          city meetups
+
+        </label>
+
+        <label>
+          
+          <input type="checkbox" value="front end project" class="input-checkbox"/>
+          Wikki
+
+        </label>
+
+        <label>
+          
+          <input type="checkbox" value="front end project" class="input-checkbox"/>
+          forum
+
+        </label>
+
+        <label>
+          
+          <input type="checkbox" value="front end project" class="input-checkbox"/>
+          Additional courses
+
+        </label>
+
+
+      </div>
```

Comment:
I think it would be better to use semantic Html tags in your project as this is used to improve on SEO.🤗💪

What do you think about that